include/inviwo/core/datastructures/transferfunction.h

```diff
@@ -134,6 +134,9 @@ class IVW_CORE_API TransferFunction : public TFPrimitiveSet {
 
     virtual std::string getTitle() const override;
 
+
+    static std::vector<TFPrimitiveData> simplify(const std::vector<TFPrimitiveData>& points, double delta);
```

Comment:
Add documentation, what is delta? Percieved color difference?, intensity?, or for each rgba-component? How is opacity treated?
What would a good default value be?